app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java

```diff
@@ -115,8 +115,6 @@ else if (eventId == -1)
             finish();
         else {
             fragmentNavigator.back();
-            binding.navView.getMenu().findItem(R.id.nav_dashboard).setChecked(true);
```

Comment:
What happens when back button is pressed from Attendees fragment. Does the Dashboard item get highlighted and title of toolbar changes to Dashboard?

Reply:
Title of toolbar is changing to Dashboard but Dashboard item is not getting highlighted. I'll fix it.